modules/loader.js

```diff
@@ -846,86 +842,92 @@ __private.snapshotFinished = err => {
  * @todo Add description for the params
  */
 __private.loadBlocksFromNetwork = function(cb) {
-	let errorCount = 0;
+	// Number of failed attempts to load from peers.
+	let tries = 0;
```

Comment:
Can we reset this when an attempt is successful? Also, the variable name can be more descriptive.